template-parts/home-page-banner.php

```diff
@@ -0,0 +1,21 @@
+<?php if( get_field("enable_home_page_title") && get_field("home_page_title")
+          && get_field("home_page_image") ): ?>
+
+  <div class="row ml-4 mr-4 pb-4 home-Page-Banner" style="background-image: url('<?php the_field('home_page_image'); ?>')" >
+
+    <div class="col md-6">
+      <div class ="home-Page-Banner-Bottom-Left pl-4">
+        <h4 class="font-weight-bold text-white overlay home-page-banner-width-title bg-dark pl-3 pr-3 pb-3 pt-3"> <?php the_field("home_page_title"); ?> </h4>
```

Comment:
This gives you the same result.
```suggestion
        <h4 class="font-weight-bold text-white overlay home-page-banner-width-title bg-dark p-3"> <?php the_field("home_page_title"); ?> </h4>
```